src/io/tree.cpp

```diff
@@ -41,6 +42,7 @@ Tree::Tree(int max_leaves, bool track_branch_features)
   leaf_value_[0] = 0.0f;
   leaf_weight_[0] = 0.0f;
   leaf_parent_[0] = -1;
+  internal_parent_[0] = -1;
```

Comment:
how do the changes here and in `tree.h` relate to the R package?

Reply:
Sorry, this was a remnant of the old version I branched off. I have removed this code now.

Reply:
got it, thanks!